jenkins/ci.suse.de/pipelines/socok8s-airship-from-rpm.Jenkinsfile

```diff
@@ -0,0 +1,181 @@
+pipeline {
+
+    options {
+        timestamps()
+        parallelsAlwaysFailFast()
+        // Note(jhesketh): Unfortunately we can't set a global timeout for the
+        //                 pipeline as it would also apply to the post stages
+        //                 and hence interrupt our cleanup.
+    }
+
+    agent {
+        node {
+            label "cloud-ccp-ci"
+        }
+    }
+
+    stages {
+        stage('Set up container to run socok8s from') {
+            steps {
+                script {
+                    // Setup for worker node (already applied)
+                    // zypper addrepo https://download.opensuse.org/repositories/Virtualization:/containers/openSUSE_Leap_15.0/ Virtualization:containers
+                    // zypper --gpg-auto-import-keys refresh
+                    // zypper install -yl podman
+                    // usermod --add-subuids 65536-362144 jenkins
+                    // usermod --add-subgids 65536-362144 jenkins
```

Comment:
Why is this block commented?

Reply:
As per the first line, this is what is required to set up the jenkins worker as a once off. It is left here as documentation on how to set up a new node if the following `which podman` were to fail.

Reply:
Thanks. Could you please update the comment so it is clearer? `(already applied)` did not make it clear for me

Reply:
Could it also be wrapped in some `if initial_jenkins_worker_setup` block?

Reply:
That would require the jenkins user to become root. I just applied these manually. If there is a better place to document the worker setup, please let me know.

Reply:
Some jenkins workers are set up with https://gitlab.suse.de/cloud/cloudsalt/ however see https://jira.suse.com/browse/SOC-9376 .

Reply:
Yes, ideally (in the future) this should go into some config management for the node. Our current CCP test node is only set up by hand though.